apidef/oas/middleware.go

```diff
@@ -1116,12 +1120,20 @@ func (tm *TransformRequestMethod) ExtractTo(meta *apidef.MethodTransformMeta) {
 // TransformBody holds configuration about request/response body transformations.
 type TransformBody struct {
 	// Enabled activates transform request/request body middleware.
+	//
+	// Tyk classic API definition: `version_data.versions.extended_paths.transform[].disabled` (negated).
```

Comment:
Nothing to action on, just updating the context 
`version_data` is [deprecated](https://github.com/TykTechnologies/tyk/blob/master/apidef/api_definitions.go#L713), however imo it makes sense to refer back to `version_data` as classic API definition users would still be relying on `version_data`.